guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -184,54 +184,58 @@
    "cell_type": "markdown",
```

Comment:
<p>I think these commands will change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
@rohitgeo should we use the commands that angad has tested and later on we can update the commands from esri channel.